recipes/bdgenomics.adam/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "bdgenomics.adam" %}
+{% set version = "0.31.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c41aeeed6deb1a5a9a20d110b15ee4e9fabdbe5c2bd05858e6b0499ae8ee3bd0
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pyspark >=1.6.0
+    - python
+  run:
+    - pyspark >=1.6.0
+    - python
+
+test:
+  imports:
+    - bdgenomics
+    - bdgenomics.adam
```

let's see if ``pip check`` will work fine here as well
```suggestion
    - bdgenomics.adam
  commands:
    - pip check
  requires:
    - pip
```